R/zzz.R

```diff
@@ -3,6 +3,7 @@ registered_handlers_namespaces <- new.env()
 
 .onLoad <- function(libname, pkgname) { # nolint
   # Set up the teal logger instance
+  teal_logger_formatter()
```

Comment:
I think that this should be part of `register_logger`
there we initiate logger and register everything it needs

Reply:
I was thinking about this. Not sure where it should be. Theoretically `teal_logger_formatter` doesn't set any "namespace" value, so it doesn't need to be re-called in every package.

Reply:
you are right - it is independent on namespace
my thinking was more like: why do I set up formatter if I don't have logger registered
moreover, `register_logger` nicely encapsulates all the logger init calls and I do consider this as a part of init process

Reply:
> my thinking was more like: why do I set up formatter if I don't have logger registered

You would have `logger` registered because `teal.logger` have `.onLoad({register_logger("teal.logger")})`. 
In the current implementation `log_formatter` is called once, globally for all packages, while your proposition is to call the same `log_formatter` in each package via `register_logger(<namespace>)` function. It is not a big performance problem.

> register_logger nicely encapsulates all the logger init calls and I do consider this as a part of init process

I have no strong opinion if it should be in `.onLoad` or in `register_logger(<namespace>)`. What do you think given the above answer?

Reply:
yes good points. Let's leave it then